server/src/main/java/org/opensearch/search/profile/aggregation/AggregationProfileBreakdown.java

```diff
@@ -62,4 +63,18 @@ public void addDebugInfo(String key, Object value) {
     protected Map<String, Object> toDebugMap() {
         return unmodifiableMap(extra);
     }
+
+    /**
+     * Build a timing count startTime breakdown for aggregation timing types
+     */
+    @Override
+    protected final Map<String, Long> buildBreakdownMap(AbstractProfileBreakdown<AggregationTimingType> breakdown) {
+        Map<String, Long> map = new HashMap<>(breakdown.timings.length * 3);
+        for (AggregationTimingType timingType : breakdown.timingTypes) {
+            map.put(timingType.toString(), breakdown.timings[timingType.ordinal()].getApproximateTiming());
+            map.put(timingType + "_count", breakdown.timings[timingType.ordinal()].getCount());
+            map.put(timingType + "_startTime", breakdown.timings[timingType.ordinal()].getEarliestTimerStartTime());
```

Comment:
lets add the static constants for `_count` and `_startTime` suffix in `AbstractProfileBreakdown` and use that here and in `ConcurrentAggregationProfiler`. Also for `startTime` will have the convention as `start_time`

```
private static final String TIMINGTYPE_COUNT_SUFFIX = "_count";

private static final String TIMINGTYPE_START_TIME_SUFFIX = "_start_time";
```

Reply:
It has been addressed now. Thanks.